rollup.config.js

```diff
@@ -2,20 +2,47 @@ import resolve from 'rollup-plugin-node-resolve';
 import typescript from 'rollup-plugin-typescript2';
 import postcss from 'rollup-plugin-postcss';
 import { terser } from "rollup-plugin-terser";
+import { dependencies } from './package.json';
 
-const esm = {
+const mjs = {
   plugins: [
     typescript({ useTsconfigDeclarationDir: false }),
     postcss()
   ],
-  external: ['pointer-tracker'],
+  external: Object.keys(dependencies),
```

Comment:
I also went ahead and replaced this with an explicit import from `package.json`'s dependencies, since it's better future-proofing as we don't need to update this whenever our dependencies change.